### DISCUSSION TITLE
regarding bead selection in the Martini 3 force field

### DISCUSSION THREAD

**Discussion by zhangtrzhangtr** [ricalessandri/Martini3-small-molecules#56] [Q&A]
![1721397488210](https://github.com/user-attachments/assets/3db8b0a9-8abb-4625-9841-728261bc5555)
Is it reasonable to use two TX4e beads and one TC5 bead to construct a model for a benzene ring substituted with two fluorine atoms when initially establishing the model ? Thank you for your help.

**Comment by ricalessandri** [comment]
@zhangtrzhangtr I think so, even though the X4e beads have not been extensively validated. We had only a few examples in our "training" set during Martini 3 parametrization, and I think exclusively molecules containing only one such bead. Nevertheless, that'd be my first guess. So, similarly to my answer on https://github.com/ricalessandri/Martini3-small-molecules/discussions/55, I recommend to get your hands on the exp. logP data for 1-2-difluorobenzene (this time it should be in this paper https://www.sciencedirect.com/science/article/abs/pii/037842749503356P), compute logP for the TC5-TX4e-TX4e CG model, and compare to make sure the bead type choice is appropriate.
